Tests/AppTests/API+PackageController+GetRoute+ModelTests.swift

```diff
@@ -119,84 +116,29 @@ class API_PackageController_GetRoute_ModelTests: SnapshotTestCase {
                                                                        platformBuildInfo: nil,
                                                                        weightedKeywords: [],
                                                                        swift6Readiness: nil,
-                                                                       forkedFromResult: nil
-                                                                      ))
+                                                                       forkedFromInfo: nil))
 
         // validate
         XCTAssertEqual(model.documentationTarget, .external(url: "https://example.com/package/documentation"))
     }
     
-    func test_init_forked_from_github() async throws {
-        let pkg = try await savePackage(on: app.db, "1".url)
-        let repo = try Repository(package: pkg, forkedFrom: .parentURL("https://github.com/example/repo.git"), name: "bar", owner: "foo")
-        try await repo.save(on: app.db)
-        let version = try App.Version(package: pkg, latest: .defaultBranch, packageName: nil, reference: .branch("main"))
-        try await version.save(on: app.db)
-        let packageResult = try await PackageResult.query(on: app.db, owner: "foo", repository: "bar")
-        let forkedFromResult = try await API.PackageController.GetRoute.fetchForkedFromResult(on: app.db, repository: packageResult.repository)
-
-        // MUT
-        let model = try XCTUnwrap(API.PackageController.GetRoute.Model(result: packageResult,
-                                                                       history: nil,
-                                                                       products: [],
-                                                                       targets: [],
-                                                                       swiftVersionBuildInfo: nil,
-                                                                       platformBuildInfo: nil,
-                                                                       weightedKeywords: [],
-                                                                       swift6Readiness: nil,
-                                                                       forkedFromResult: forkedFromResult
-                                                                      ))
-
-        // validate
-        XCTAssertEqual(model.forkedFromInfo, API.PackageController.GetRoute.Model.ForkedFromInfo.fromGitHub(url: "https://github.com/example/repo.git"))
-    }
```

Comment:
This test is now pretty much superfluous - we're passing through the simple `ForkedFromInfo` enum and there's no point testing the GH case.